src/config/routes.ts

```diff
@@ -30,9 +30,19 @@ export const relativeRoutes = {
     editChannel: () => 'channel',
     videos: () => 'videos',
     uploads: () => 'uploads',
-    signIn: () => 'signIn',
+    signIn: () => 'signin',
+    join: ({ step }: { step?: string } = {}) => {
```

Comment:
since this function does the same thing as the one in search route maybe you could extract these into one that is generic?